### ISSUE TITLE
Excel import error after updating to the new version 3.1.7

### ISSUE THREAD

**Issue by muzhawir** [eighty9nine/filament-excel-import#31]
After updating to the new version, an error appears when I click the import button:

![image](https://github.com/user-attachments/assets/5b2c1430-e472-4925-a4f4-70412b159453)

This code is in the `ListEmployees` class, as shown in the README example, here is my code:

```php
ExcelImportAction::make()
    ->afterImport(
        fn () => Notification::make()
            ->success()
            ->title('Impor data dari Excel berhasil!')
            ->send()
    )
    ->color('primary')
    ->icon('heroicon-o-document-text')
    ->label('Impor Excel')
    ->modalHeading('Impor Data Dari Excel')
    ->slideOver()
    ->use(EmployeeImport::class)
```

This is the error log from the `clockwork` package related to the issue:

![image](https://github.com/user-attachments/assets/78d3c88e-951b-4957-8be6-15ccbb14fb8e)

**Comment by eighty9nine** [comment]
This error has been fixed in v3.1.8

**Comment by muzhawir** [comment]
Thank you!
